suite/tests/CMakeLists.txt

```diff
@@ -3029,15 +3029,15 @@ if (X86 OR AARCH64)
   endif ()
 endif ()
 
-if (NOT RISCV64)
+#if (NOT RISCV64)
```

Comment:
Delete it instead of commenting it out, same below.

Reply:
Done